### ISSUE TITLE
error 'LOG_OUT', code=8

### ISSUE THREAD

**Issue by Chriskatz** [merkremont/LineVodka#18]
LineAlpha.LineThrift.ttypes.TalkException: TalkException(parameterMap=None, reason='LOG_OUT', code=8)
Seem start blocking.

**Comment by doodspav** [comment]
yup getting this rn - ill tell u if i figure it out (btw for anyone who didnt figure out the previous ones, line depreciated mac headers)

**Comment by Dewamade** [comment]
@doodspav how do you figure it out?, can you please share your method?, thanks

**Comment by Sanguarian** [comment]
File "vodkabot.py", line 12, in <module>
    client._qrLogin("line://au/q/")
  File "/home/cabox/workspace/LineVodka/LineAlpha/LineApi/LineApi.py", line 110, in _qrLogin
    self.onLogin()
  File "/home/cabox/workspace/LineVodka/LineAlpha/LineApi/LineApi.py", line 37, in onLogin
    self.revision = self._client.getLastOpRevision()
  File "/home/cabox/workspace/LineVodka/LineAlpha/LineThrift/TalkService.py", line 3467, in getLastOpRevision
    return self.recv_getLastOpRevision()
  File "/home/cabox/workspace/LineVodka/LineAlpha/LineThrift/TalkService.py", line 3490, in recv_getLastOpRevision
    raise result.e
LineAlpha.LineThrift.ttypes.TalkException: TalkException(parameterMap=None, reason='LOG_OUT', code=8)

**Comment by Sanguarian** [comment]
previous to this i started up the one i got working last night with the authtoken already in it it was functional (until i deleted it) but that being said.... could this be something with the qrLogin, maybe something was changed there

**Comment by semarainc** [comment]
i got the same issue
but how can i get the new UA&LA?

**Comment by cosadiz69** [comment]
this thing can be overcome

**Comment by fpitlok** [comment]
I know the reason is because LINE brings the certificate encryption. 

**It's easy to fix**  
you can find a CHOME user-agent to use, because CHOME has a valid certificate.

**But** It does not use the correct solution, because the real problem is "SSL" encryption.

I can only say this.
good luck :blush:
